ethereum/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "ethereum"
```

Comment:
Better to use another name for this crate

Reply:
Yes

Reply:
How about ethereum-chains for the crate name?